source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,211 @@
+
+What can you customize?
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can customize your sidebar in the following ways:
+
+- Group and order channels into collapsible categories you create.
+- Sort each channel in a category manually, alphabetically, or by recent activity.
+- Mute and unmute entire categories.
+- Filter your sidebar to view unread channels only, or choose to group unread messages into an **Unreads** category.
+- Sort Direct Messages alphabetically or by recent activity, and choose how many are displayed in your sidebar.
+
+  .. image:: ../../images/channel_sidebar_updates.gif
+
+Creating custom categories
+--------------------------
+
+Create custom categories to group channels together for quicker and easier navigation (e.g. “Design” or “Marketing”). For example, you can create a category called "Design" or "Marketing".
+
+Categories can be created in two ways:
+
+- Select the **+** symbol at the top of the sidebar.
+
+Or
+
+- Select the **More options...** icon in the sidebar on any category header, then select **Create New Category**.
+
+Next, type a category name, select **Create**, then drag any channels or Direct Messages into this new category. You can also multi-select channels and Direct Messages to drag them together as a group using CTRL/CMD or SHIFT. See **Dragging and dropping selections** below for details.
+
+Making categories work for you
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Categories are collapsible**
+
+- When collapsed, only unread channels display to reduce unnecessary scrolling.
+- When expanded, all channels in the category display, including channels with unread messages.
+
+**You can reorder categories**
+
+- Drag to reorder entire categories to prioritize important conversations. 
+
+**Categories can contain Direct Message conversations**
+
+- Select and drag Direct Messages into any category. You can also multi-select Direct Messages to drag them together as a group.
+
+Renaming categories
+^^^^^^^^^^^^^^^^^^^
+
+1. Select the **Category options** icon in the sidebar, then select **Rename Category**.
+2. Type a new category name, then select **Rename**.
+
+Deleting categories
+^^^^^^^^^^^^^^^^^^^
+
+1. Select the **Category options** icon in the sidebar, then select **Delete Category**.
+2. Select **Delete** to confirm or select **X** to cancel.
+
+All channels and Direct Message conversations in the deleted category move back to their default **Channels** and **Direct Messages** categories. Deleting a category never removes you from channels you have joined. 
+
+Organizing channels in categories
+---------------------------------
+
+Once you've created categories, you can move channels within them to organize your sidebar.
+
+Dragging and dropping selections
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the Mattermost Web or Desktop App, drag selected channels and/or Direct Messages between or within categories. 
+
+.. tip::
```

Comment:
I'd propose the three bullets and the subsequent sentence are a tip together. Perhaps that was the intention but the syntax got messed up

```
Multi-selected channels and Direct Messages move together as a group in the order they originally appeared.

- Select sequential channels and/or Direct Messages by pressing and holding SHIFT. 
- Select non-sequential channels and/or Direct Messages by pressing and holding CMD (for Mac) or CTRL (for Windows/Linux).
- Press ESC to clear channel or Direct Message selections.
```

I also think that it needs to be clear that you have to "click". ie Shift + Click, CMD + Click, so propose we work that into the above wording

![image](https://user-images.githubusercontent.com/14333569/106948787-01466f00-66e1-11eb-84ca-39a664c63732.png)